ground/src/main/java/com/google/android/ground/persistence/local/LocalValueStore.kt

```diff
@@ -45,11 +45,11 @@ class LocalValueStore @Inject constructor(private val preferences: SharedPrefere
       preferences.edit().putInt(MAP_TYPE, type).apply()
     }
 
-  /** State of the gps lock. */
-  var isGpsLocked: Boolean
-    get() = preferences.getBoolean(GPS_LOCKED, false)
+  /** State of the location lock. */
+  var isLocationLocked: Boolean
```

Comment:
"Location locked" implies the location cannot change; we're not locking the location, we're enabled the "location lock" feature, thus "location lock enabled/disabled" makes more sense here.

Reply:
Done